how-not-to-be-a-dinosaur.md

```diff
@@ -0,0 +1,117 @@
+
+_Nope. Not me either._
+
+And after a year of doing (UX) design, I found out this was the wrong choice. Years of intensive therapy and running into a black hole that was another potential burnout, I came to the conclusion that design was not for me. I can design, but I can't handle the stress that goes with it. 
+
+And then it was late 2018. I decided to get back to frontend development.
+
+![](https://raw.githubusercontent.com/MyOnlineStore/blogs/Dinosaurusblog/public/johan_dino.png)
+
+_Yes this is me._
+
+## How to get up to speed? 
+
+Now I faced a big challenge. How to get up to speed after being paused for a few years in a world that changed (and still is changing) so fast? First thing to overcome was my aversion for The New Frontend World. I tried to look at React and all new tools with a fresh and unbiased view.
+
+While I was away in design world a lot has changed within our codebase, but also in The Frontend World. The tools we used improved or were replaced with better alternatives. The tools and codebase became more solid and mature.
+
+Then there was news. React Hooks was about to be released. Which made code só much more readable that I started thinking "Hmm, maybe I can do this".  
+
+After this "challenge accepted!" point the hardest obstacle had to be overwon: how to handle the "I have to start all over again" feeling. I felt like I knew nothing. I was Jon Snow.
+
+## Dem feelings...
+
+Who had I been kidding all these years calling myself a senior developer? How emberrassing was it that I had to ask my 12 years younger colleague for help over and over again...  
+
+![](https://raw.githubusercontent.com/MyOnlineStore/blogs/Dinosaurusblog/public/johan_knowsnothing.png)
+
+Some people will think "what's the big deal?", and I am happy for you if you do, but you have to understand that asking help was a big issue for me. And a major cause for my burn out and personality disorder that came with it.
+
+So I did two things: I decided to start learning all these new tools and code, but more importantly I started to ask for help. And the best thing I did: I told my colleagues how I was feeling and how insecure I was about everything. They responded very well to this and this gave me the confidence to go for it.
+
+Also, I realised I did not have to start all over again. It might have felt like it, but I still had a big headstart with the years of experience in programming overall. It's just a new language and new tools I needed to learn, not learning how to speak and not the first tool I ever used.
+
+It will take some time to learn all these new things. And that's okay. Good things take time. I should have realised this earlier since it's litteraly written on my body, but sadly it's not that obvious sometimes.
+
+![](https://raw.githubusercontent.com/MyOnlineStore/blogs/Dinosaurusblog/public/johan_arm.png)
+
+_Yes, this is also me._
+
+## … okay great, no fucks given, but what then?
```

Comment:
Same here as with the title, fucks have been given but you have conquered the fucks! This could be more positive, like:

```suggestion
## Okay great, challenge accepted! Where to begin?
```